index.html

```diff
@@ -183,7 +251,267 @@ <h4 class="ring"><a href="#" class="over">Wolt</a></h4>
 </div>
 
 <br>
+<div class="Categories">
```

Comment:
Kindly replace `<div class="Categories">` to `<section class="Categories">`, so we improve the use of HTML semantics. Kindly do this for the others as well